static/scss/sort-picker.scss

```diff
@@ -1,7 +1,7 @@
-.sorter {
+.picker {
   cursor: pointer;
 
-  .current-sort {
+  .current-pick {
     display: flex;
     align-items: center;
   }
```

Comment:
might want to rename this scss file while we're at it

Reply:
Ah right